apps/web/src/components/message-center-handler/index.tsx

```diff
@@ -36,9 +37,16 @@ export function MessageCenterHandler({ children }: { children?: ReactNode }) {
           await router.push('/');
           router.reload();
         }
+
+        if (message.code === MessageCenter.messageCode.refreshTokenError) {
```

Comment:
It seems `refreshTokenError` and `noPermission` errors are handled in the same way right now (other than the error toast). Maybe we can merge the codes?

Reply:
RefreshTokenError is handled as a temporary and quick solution, with complex logic added later